packages/config/src/__mocks__/Config.ts

```diff
@@ -0,0 +1,57 @@
+/*
```

Comment:
I can move them around 😋 
Should we also move `imperative/src/__mocks__` and `utilities/src/__mocks__` ?

Reply:
IDK if this will be confusing to developers though...

```typescript
// jest mocks
jest.mock("../../../../utilities/ImperativeConfig");
jest.doMock("../../../../utilities/ImperativeConfig");

// requires
let {ImperativeConfig} = require("../../../../utilities/src/ImperativeConfig");
// imports
({ImperativeConfig} = await import("../../../../utilities/src/ImperativeConfig"));
```

Reply:
It should be fine, I was concerned about having mocks inside the "src" folder but didn't realize we already do this elsewhere and exclude all "__mocks__" in our tsconfig.json 👍